codalab/lib/cli_util.py

```diff
@@ -109,6 +109,12 @@ def get(dep):  # Return the key
                     'key %s exists with multiple values: %s and %s' % (key, key2val[key], val)
                 )
         else:
+            if key is None:
+                raise UsageError(
+                    'Dependency is empty. Please provide a valid dependency in the format of [<key>]:'
+                    '[[(<alias>|<address>)::](<uuid>|<name>)//](<uuid>|<name>|^<index>)[/<subpath within bundle>].'
```

Comment:
Can you not duplicate this here but instead use the `*_SPEC_FORMAT` variables defined in `bundle_cli.py`?  You might have to move those definitions.

Reply:
Updated.